gui/saveload-dialog.cpp

```diff
@@ -483,28 +484,42 @@ void SaveLoadChooserSimple::reflowLayout() {
 		if (textLines > 0)
 			textLines++; // add a line of padding at the bottom
 
-		_container->resize(x, y, w, h + (kLineHeight * textLines));
-		_gfxWidget->resize(thumbX, thumbY, thumbW, thumbH);
+		if (_thumbnailSupport) {
+			_gfxWidget->resize(thumbX, thumbY, thumbW, thumbH);	
+			_gfxWidget->setVisible(true);
+		}
+		else { // choose sensible values for displaying playtime and date/time when a thumbnail is not being used
```

Comment:
Our [code formatting conventions](http://wiki.scummvm.org/index.php/Code_Formatting_Conventions#Hugging_braces) indicate the else should not be on a new line (and same thing for the other two else below.